build.yaml

```diff
@@ -452,6 +455,13 @@ steps:
    publishAs: benchmark
    dependsOn:
      - service_base_image
+ - kind: buildImage
+   name: monitoring_image
+   dockerFile: monitoring/Dockerfile
+   contextPath: .
```

Comment:
This docker file only depends on the contents of `monitoring`. Let's be specific about that.

Reply:
I don't understand this comment. We need functions in hailtop and gear in this Docker image.